bedrock/mozorg/templates/mozorg/home/home-de.html

```diff
@@ -49,7 +49,7 @@ <h1>{{ self.page_title() }}</h1>
       </div>
 
       {{ billboard(
-        title='Für Dich & das Internet. <br>Schon seit 1998.',
+        title='Für Dich & das Internet. <br>Schon seit 1998.'|safe,
```

Comment:
This is just a fix for a bug I spotted, unrelated to this PR (the `<br>` was rendered as text, not sure how long this bug has been in production).